### PR TITLE
1872 Corrected 'Error' message for FTP download

### PR DESCRIPTION
**Pull request by DaveBlueFrontier**
when no files to process.

### CODE REVIEW

File: app/models/ftp_monitor.rb [hunk 1]
```diff
@@ -42,7 +42,11 @@ def process!
       Rails.logger.info("Listed files from #{ftp_info.hostname}: #{files.join(', ')}")
       # Remove files already seen
       files -= already_reviewed_files
-
+      # Return if there are no more files
+      if files.length <= 0
+        Rails.logger.info("No files to process from #{ftp_info.hostname}")
```
Comment (tripledub) [inline app/models/ftp_monitor.rb]: `I18n`?

Comment (tripledub) [inline app/models/ftp_monitor.rb]: i.e. `Rails.logger.info(I18n.t('ftp.no_files_to_process'))`

Comment (DaveBlueFrontier) [inline app/models/ftp_monitor.rb]: Good idea.  I'll have to do the lot of 'em though - can't have just one being done.

Comment (tripledub) [inline app/models/ftp_monitor.rb]: We agreed at StandUp to do them as we come to them from now on

Comment (DaveBlueFrontier) [inline app/models/ftp_monitor.rb]: It's good that we have all the details on hand for how to implement them then :)

Comment (tripledub) [inline app/models/ftp_monitor.rb]: Well.... My next PR has an example - otherwise you only have to ask

**Pull request merged**